api/devices/schema.py

```diff
@@ -156,3 +180,9 @@ class Mutation(graphene.ObjectType):
             \n- location: The location of the device\
             \n- room_id: Unique identifier of a room where the device is found\
             [required]")
+    delete_device = DeleteDevice.Field(
+        description="Updates a given device details given the arguments to delete\
```

Comment:
This description is not correct, I think it is supposed to `delete` not `update`, accepts only one argument `device_id` unique identifier of a `device`

Reply:
@mnswaleh : I think i am  updating the state column changing from active to archived or delete so i think the description is accurate if not close enough

Reply:
@mnswaleh : resolved